src/components/pod/pod.test.js

```diff
@@ -3,7 +3,52 @@ import Pod from "./pod.component";
 import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
 import Content from "../content";
 
-import { podBlock } from "../../../cypress/locators/pod";
+import {
+  podBlock,
+  podComponent,
+  podTitle,
+  podSubTitle,
+  podFooter,
+  podSoftDelete,
+  podContent,
+  podEditContainer,
+  podDelete,
+  podUndo,
+  podEdit,
+} from "../../../cypress/locators/pod";
+
+import { useJQueryCssValueAndAssert } from "../../../cypress/support/component-helper/common-steps";
+
+const specialCharacters = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+
+const PodComponent = ({ ...props }) => {
+  return (
+    <Pod
+      title="Title"
+      subtitle="Subtitle"
+      footer="Footer"
+      onEdit={() => {}}
+      onDelete={() => {}}
+      mb={3}
+      {...props}
+    />
+  );
+};
+
+const SoftDeletePodComponent = ({ ...props }) => {
+  return (
+    <Pod
+      title="Title"
+      subtitle="Subtitle"
+      description="Description"
+      footer="Footer"
+      onUndo={() => {}}
+      {...props}
```

Comment:
```suggestion
      softDelete
      {...props}
```
Personally I think we should add this prop by default, since the `Pod` will not render with the soft delete styles unless both `softDelete` and `onUndo` defined. This should not affect the test on line 165, as we are spreading the props on line 46 which will overwrite any prop values.